sources/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -39,6 +39,7 @@ do_upgrade() {
     nspawn_opts="--capability=all --bind=/sys --bind=/dev --bind=/proc --bind=/run/udev --keep-unit --register=no"
     $NEWROOT/bin/systemd-nspawn $nspawn_opts -D $NEWROOT $LEAPPBIN upgrade --resume $args
     rv=$?
+    sync
```

Comment:
Add at least some comment to this so people won't be wondering why it's here when looking at it in a year from now.